src/style.css

```diff
@@ -1,3 +1,77 @@
 body {
   margin: 0;
+
+}
+
+.logo__img {
+  padding: 10px 0;
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  font-size: 12px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  display: block;
+
+}
+
+.nav__list {
+  margin: 0;
+  list-style: none;
+  padding: 0;
```

Comment:
Separate `reset type` styles from the rest by moving them to the top of the file. Later such styles will lay in separate file.